src/main/java/frc/robot/joystickManager/JoystickBindings.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot.joystickManager;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.Robot;
+import frc.joysticks.SmartJoystick;
+
+
+public class JoystickBindings {
+
+	public static void configureBindings(SmartJoystick joystick, Robot robot) {}
```

Comment:
I still dont understand how do you bind a command like swerve command which needs to have a reference to the joystick inside it